testsuite/integration-tests-spring/deployment/src/test/java/org/jboss/resteasy/test/spring/deployment/AddSpringResteasyAsResourceRootTest.java

```diff
@@ -103,6 +118,16 @@ public void testDeploymentWithSpringContextLoaderListenerAddsResourceRoot() thro
                 .addClass(TestResource.class)
                 .addAsWebInfResource(AddSpringResteasyAsResourceRootTest.class.getPackage(), "web.xml", "web.xml")
                 .addAsWebInfResource(AddSpringResteasyAsResourceRootTest.class.getPackage(), "applicationContext.xml", "applicationContext.xml");
+
+        archive.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
+            new PropertyPermission("arquillian.*", "read"),
+            new ReflectPermission("suppressAccessChecks"),
+            new RuntimePermission("accessDeclaredMembers"),
+            new RuntimePermission("getClassLoader"),
+            new FilePermission("<<ALL FILES>>", "read"),
+            new LoggingPermission("control", "")
+        ), "permissions.xml");
```

Comment:
I can not reproduce this.  All the listed permission are required.
I added a comment to the permission block.